core/templates/dev/head/AppInit.ts

```diff
@@ -16,11 +16,39 @@
  * @fileoverview File for initializing the main oppia module.
  */
 
+import { NgModule } from '@angular/core';
+import { BrowserModule } from '@angular/platform-browser';
+import { StaticProvider } from '@angular/core';
+
+@NgModule({
+  imports: [
+    BrowserModule
+  ]
+})
+class MainAngularModule {
+  // Empty placeholder method to satisfy the `Compiler`.
+  ngDoBootstrap() {}
+}
+
+import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
+import { downgradeModule } from '@angular/upgrade/static';
+
+const bootstrapFn = (extraProviders: StaticProvider[]) => {
+  const platformRef = platformBrowserDynamic(extraProviders);
+  return platformRef.bootstrapModule(MainAngularModule);
+};
+const downgradedModule = downgradeModule(bootstrapFn);
+
+declare var angular: any;
+
 var oppia = angular.module(
   'oppia', [
     'dndLists', 'headroom', 'infinite-scroll', 'ngAnimate',
     'ngAudio', 'ngCookies', 'ngImgCrop', 'ngJoyRide', 'ngMaterial',
     'ngResource', 'ngSanitize', 'ngTouch', 'pascalprecht.translate',
-    'toastr', 'ui.bootstrap', 'ui.sortable', 'ui.tree', 'ui.validate'
+    'toastr', 'ui.bootstrap', 'ui.sortable', 'ui.tree', 'ui.validate',
+    downgradedModule
   ].concat(
   window.GLOBALS ? (window.GLOBALS.ADDITIONAL_ANGULAR_MODULES || []) : []));
+
+exports.moduleName = oppia;
```

Comment:
Can we rename this to `exports.module` since we are not exporting the name but the whole module.

Reply:
Done